src/registries/npm-registry.js

```diff
@@ -14,7 +14,7 @@ import {addSuffix} from '../util/misc';
 import {getPosixPath, resolveWithHome} from '../util/path';
 
 const normalizeUrl = require('normalize-url');
-const userHome = require('../util/user-home-dir').default;
+const {default: userHome, home} = require('../util/user-home-dir');
```

Comment:
While you're here, could you please change this to an `import` statement? I've been meaning to revisit one of my old pull requests where I updated all `require` statements to use `import`, but just haven't had time.